packages/lab/src/carousel/Carousel.tsx

```diff
@@ -1,170 +1,47 @@
-import {
-  Button,
-  GridLayout,
-  RadioButton,
-  RadioButtonGroup,
-  makePrefixer,
-  useIcon,
-  useId,
-} from "@salt-ds/core";
-import { useComponentCssInjection } from "@salt-ds/styles";
-import { useWindow } from "@salt-ds/window";
-import { clsx } from "clsx";
-import {
-  type ChangeEventHandler,
-  Children,
-  type HTMLAttributes,
-  type ReactElement,
-  forwardRef,
-  useEffect,
-} from "react";
-import { DeckLayout } from "../deck-layout";
-import { useSlideSelection } from "../utils";
-import type { CarouselSlideProps } from "./CarouselSlide";
-
-import carouselCss from "./Carousel.css";
-
-const withBaseName = makePrefixer("saltCarousel");
+import { type HTMLAttributes, forwardRef } from "react";
+import { CarouselProvider } from "./CarouselContext";
 
 export interface CarouselProps extends HTMLAttributes<HTMLDivElement> {
   /**
    * The initial Index enables you to select the active slide in the carousel.
    * Optional, default 0.
    **/
-  initialIndex?: number;
-  /**
-   * The animation when the slides are shown.
-   * Optional. Defaults to `slide`
-   **/
-  animation?: "slide" | "fade";
+  activeSlideIndex?: number;
   /**
    * If this props is passed it will set the aria-label with value to the carousel container.
    * Optional. Defaults to undefined
    */
   carouselDescription?: string;
```

Comment:
We should probably just use aria-label in this case

Reply:
Just thought a bit more and I think the title in the controls will be programatically linked to the carousel so this won't be needed most of the time anyway